packages/jds/src/components/Uploader/uploader.utils.ts

```diff
@@ -0,0 +1,6 @@
+export const validateAcceptedFile = (accept: string[], file: File) => {
+  return accept.some(type => {
+    if (type.startsWith('.')) return file.name.endsWith(type);
```

Comment:
```suggestion
    if (type.startsWith('.')) return file.name.endsWith(type.toLowerCase());
```

확장자가 대문자인 경우를 고려한다면 이런식으로 처리해도 될 거같네요. 
(endsWith의 경우 대소문자를 구분하기 때문)